### DISCUSSION TITLE
The page number in the toolbar is not updating correctly.

### DISCUSSION THREAD

**Discussion by devwitalik** [sumatrapdfreader/sumatrapdf#3757] [General]
I use the assembly: SumatraPDF-prerel-64-install_15699 (you can use any of the latest builds, the problem will persist)

The sequence of reproducing the error:

Open a document (fb2, pdf, document format does not matter)

Start reproducing the issue from any page of the open document.

Next step: Disable the Toolbar display (hotkey: F8)

Next step: Use the right or left mouse button to scroll through the document by clicking outside the document itself, namely on the MainWindowBackground (the sidebar outside the document itself). Move the pages down (or up) with the mouse in the manner described above for at least three pages. When you want to stop turning pages, (IMPORTANT) leave a small part of the previous page visible.

Next step: Press the G hotkey to open the Go to page window, which displays the correct page number.
Next step: Turn on the display of the Toolbar (hotkey: F8)

Check that the correct page number is displayed in the Toolbar ?

**Comment by GitHubRulesOK** [comment]
Hmm 
Yes thats one way to upset the page indicators until next triggered count, but the shifted I (Info) is still good

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/494bd066-3073-476b-abec-b8ae3eba1242)

@kjk seem to recollect something similar in the past, but do you want to shift to Open Issue ? or leave as an observation.
